packages/test/test-service-load/src/FileLogger.ts

```diff
@@ -93,7 +93,20 @@ class FileLogger implements ITelemetryBufferedLogger {
 		) {
 			event.category = "generic";
 		}
-		this.baseLogger?.send({ ...event, hostName: pkgName, testVersion: pkgVersion });
+
+		const plainEvent = {
+			...event,
+			hostName: pkgName,
+			testVersion: pkgVersion,
+		};
+		let augmentedEvent: ITelemetryBaseEvent | undefined;
+		if (process.env.FLUID_LOGGER_PROPS?.includes("displayName") !== false) {
+			augmentedEvent = {
+				...plainEvent,
+				details: JSON.stringify(process.env.FLUID_LOGGER_PROPS),
+			};
```

Comment:
Maybe this so we don't need the extra event object? Also, I think what we want here is `JSON.parse()` instead of `JSON.stringify()`? The env variable is a string by default, and the working example we have where we use details passes an actual JSON object. 
```suggestion
			plainEvent.details = JSON.stringify(process.env.FLUID_LOGGER_PROPS);
```